### PR TITLE
added a telegram send message in logger with log-level

### PR DESCRIPTION
**Pull request by Pirngruber**

### CODE REVIEW

**Review by lourencor — APPROVED** [review]
Seems ok and a useful functionality

**Comment by rafalsza** [comment]
tested and it's working fine

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,7 @@ config.yml
 
 # Ignore test file
 test_order.py
+
+.github/
```
Comment (DominicFrei) [inline .gitignore:25]: Did you want to ignore this one?

File: .gitignore [hunk 2]
```diff
@@ -21,3 +21,7 @@ config.yml
...
+
+.gitignore
```
Comment (DominicFrei) [inline .gitignore:27]: You probably do not want to ignore the `.gitignore` itself.

**Review by DominicFrei — COMMENTED** [review]
Added some comments about the changes in the `.gitignore`.

**Comment by DominicFrei** [comment]
Is this the same as https://github.com/CyberPunkMetalHead/gateio-crypto-trading-bot-binance-announcements-new-coins/pull/18 ?

File: logger.py [hunk 3]
```diff
@@ -14,8 +15,15 @@
 # Get logging variables
 log_level = config['LOGGING']['LOG_LEVEL']
 log_file = config['LOGGING']['LOG_FILE']
-log.basicConfig(format='%(asctime)s %(levelname)s: %(message)s',
+send_telegram = config['TELEGRAM']['SEND_MESSAGES']
+
+if(send_telegram):
+    log.basicConfig(format='%(asctime)s %(levelname)s: %(message)s',
+                handlers=[logging.FileHandler(log_file), logging.StreamHandler(), TelegramHandler()])
```
Comment (DominicFrei) [inline logger.py:22]: @HebelHuber This might be a good solution too. Having them all as handlers in the logger.

**Comment by DominicFrei** [comment]
https://github.com/CyberPunkMetalHead/gateio-crypto-trading-bot-binance-announcements-new-coins/pull/18 covers this one, closing it for now. If someone sees something missing, please feel free to comment. :+1:

**Pull request closed without merging**